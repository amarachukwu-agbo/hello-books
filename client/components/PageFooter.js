import React from 'react';

const PageFooter = () => (
    <footer className="page-footer foot lighten-5">
        <div className="footer-copyright">
            <div className="container center-align grey-text">
                Â© 2017 Copyright Hello-books
                    </div>
        </div>
    </footer>
);

export default PageFooter;
