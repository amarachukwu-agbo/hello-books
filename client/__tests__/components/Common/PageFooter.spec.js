import React from 'react';
import { shallow } from 'enzyme';
import PageFooter from '../../../components/Common/PageFooter';

describe('<PageFooter>', () => {
  it('should render the footer correctly', () => {
    const tree = shallow(<PageFooter />);
    expect(tree).toMatchSnapshot();
  });
});
