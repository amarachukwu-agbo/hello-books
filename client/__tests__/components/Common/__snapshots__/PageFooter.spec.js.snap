// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageFooter> should render the footer correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageFooter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="footer-copyright"
      >
        <div
          className="container center-align grey-text"
        >
          © 2017 Copyright Hello-books
        </div>
      </div>,
      "className": "page-footer foot lighten-5",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container center-align grey-text"
        >
          © 2017 Copyright Hello-books
        </div>,
        "className": "footer-copyright",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "© 2017 Copyright Hello-books",
          "className": "container center-align grey-text",
        },
        "ref": null,
        "rendered": "© 2017 Copyright Hello-books",
        "type": "div",
      },
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer-copyright"
        >
          <div
            className="container center-align grey-text"
          >
            © 2017 Copyright Hello-books
          </div>
        </div>,
        "className": "page-footer foot lighten-5",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container center-align grey-text"
          >
            © 2017 Copyright Hello-books
          </div>,
          "className": "footer-copyright",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "© 2017 Copyright Hello-books",
            "className": "container center-align grey-text",
          },
          "ref": null,
          "rendered": "© 2017 Copyright Hello-books",
          "type": "div",
        },
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
